packages/contracts-bedrock/test/L2/CrossL2Inbox.t.sol

```diff
@@ -101,7 +101,7 @@ contract CrossL2InboxTest is Test {
         vm.expectCall({ callee: _target, msgValue: _value, data: _message });
 
         // Look for the emit ExecutingMessage event
-        vm.expectEmit(Predeploys.CROSS_L2_INBOX);
+        vm.expectEmit(true, false, false, true, Predeploys.CROSS_L2_INBOX);
```

Comment:
For `expectEmit` let's default to using all `true`, which this suggestion is equivalent to. This is preferred as it's a stronger test, and if we every change topics or parameters we ensure everything is tested by default
```suggestion
        vm.expectEmit(Predeploys.CROSS_L2_INBOX);
```

Reply:
noted, done.